src/sentry/eventstream/kafka/backend.py

```diff
@@ -127,6 +137,10 @@ def insert(self, group, event, is_new, is_sample, is_regression,
             organization=project.organization,
         )
 
+        unexpected_tags = set([k for (k, v) in event.get_tags() if k in self.UNEXPECTED_TAG_KEYS])
+        if unexpected_tags:
+            logger.error('%r received unexpected tags: %r', self, unexpected_tags)
```

Comment:
This should really be a warning but error level makes it more likely to actually be logged and noticed